release/translate.go

```diff
@@ -159,6 +159,13 @@ func (releaseArch *Arch) toStreamArch(rel *Release) stream.Arch {
 		*/
 	}
 
```

Comment:
note to self - we should probably sort these alphabetically at some point.

Reply:
#66